test/unit/Application.test.js

```diff
@@ -15,6 +18,82 @@ const params = {
 };
 
 describe('Application', () => {
+  describe('Not valid param', () => {
+    let component;
+
+    before(() => {
+      // Not the best way to initialize this component:
```

Comment:
Why not?

Reply:
To be honest, I copied the same tests from other `describe` in `Application.test.js` as they will test the same DOMs. I left it as it was the previous dev's comment. I have the same question too.

Please see line 22
https://github.com/NYPL/staff-picks/blob/development/test/unit/Application.test.js